<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileProcessClass</class>
 <widget class="QMainWindow" name="FileProcessClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>454</width>
    <height>208</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FileProcess</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="static_All">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>421</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>总进度</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_Child">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>421</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="static_Child">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>421</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>当前处理中的文件</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_All">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>421</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>454</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="FileProcess.qrc"/>
 </resources>
 <connections/>
</ui>
